common-lib/src/main/scala/com/gu/mediaservice/model/ThrallMessage.scala

```diff
@@ -0,0 +1,88 @@
+package com.gu.mediaservice.model
+
+ import com.gu.mediaservice.lib.logging.{GridLogging, LogMarker}
+ import com.gu.mediaservice.model.leases.MediaLease
+ import com.gu.mediaservice.model.usage.UsageNotice
+ import org.joda.time.{DateTime, DateTimeZone}
+ import play.api.libs.json.{JodaReads, JodaWrites, Json, __}
+
+sealed trait ThrallMessage extends GridLogging with LogMarker {
+  implicit val yourJodaDateReads = JodaReads.DefaultJodaDateTimeReads.map(d => d.withZone(DateTimeZone.UTC))
+  implicit val yourJodaDateWrites = JodaWrites.JodaDateTimeWrites
+  def subject: () => String = () => this.getClass.getSimpleName
+  val id: String
+  val lastModified: DateTime
+  def additionalMarkers: () => Map[String, Any] = () => Map()
+
+  override def markerContents = {
+    val message = Json.stringify(Json.toJson(this)(ThrallMessage.writes))
+    Map (
+      "subject" -> subject,
```

Comment:
you're passing in a function here, I suspect this should be `subject()`? (or just make this a string as mentioned above)